x-pack/plugins/security_solution/common/detection_engine/schemas/alerts/8.3.0/index.ts

```diff
@@ -54,3 +51,17 @@ export interface EqlBuildingBlockFields830 extends BaseFields830 {
   [ALERT_GROUP_INDEX]: number;
   [ALERT_BUILDING_BLOCK_TYPE]: 'default';
 }
+
+export interface NewTermsFields830 extends BaseFields830 {
+  [ALERT_NEW_TERMS]: Array<string | number | null>;
+}
+
+export type NewTermsAlert830 = AlertWithCommonFields800<NewTermsFields830>;
```

Comment:
Should a new `8.4.0` schema be created instead of these changes applied to the `8.3.0` alerts schema?

Reply:
Ah yeah this PR has been open for a long time :sweat_smile:

Reply:
The whole 8.3.0 folder needed to be moved to 8.4.0 as the other changes in that folder were also introduced for 8.4 (but also originally intended for 8.3).